give.spec

```diff
@@ -1,24 +1,23 @@
-%define realname give
-%define realversion 3.1-5
-
-#don't strip binaries
+# The default behavior of rpmbuild is to strip binaries after installation
+# in order to reduce file size. The two macros defined below override that
+# behavior and compresses the man and info pages for debugging purposes.
 %define __os_install_post /usr/lib/rpm/brp-compress
 %define debug_package %{nil}
 
-Name: %{realname}-%{realversion}
-Version: ptools
-Release: 1
+Name: give
+Version: 3.1
+Release: 5%{?dist}
```

Comment:
increment release to 6 (yes you will have to rename your source tarball from `3.1-5.tgz` to `3.1-6.tgz` until a new tarball is cut).

Whenever you make changes to a spec file, you increment the release and provide a comment on the changelog about your changes. When a new version is cut, for example say 3.2 of give is released, then the release number would go back to 1. The release numbers indicate rpm changes.